lib/maze/bitbar_devices.rb

```diff
@@ -8,7 +9,56 @@ class BitBarDevices
     APPIUM_1_15_0 = '1.15.0'
     APPIUM_1_20_2 = '1.20.2'
 
+    BASE_URI = 'https://cloud.bitbar.com/api/v2/me'
+    FILTER_PATH = 'devices/filters'
+
+    DEVICE_GROUP_IDS = {
+      # Classic, non-specific devices for each Android version
+      'ANDROID_10_0' => '46024',
```

Comment:
I'm guessing these ids are very specific to our environment.  I started to wonder if they should be factored out into the environment (at a later stage), but I suspect that could be over-engineering it if we're not going to be changing groups around.

Reply:
I think for now it's fine, as they should be immutable within our bitbar env, but yes, it would potentially be better to load from the env at some point.